docs/development_suite/webhooks-and-events/events.mdx

```diff
@@ -100,21 +333,103 @@ hence there is no guarantee that the service will be found in the project config
 | `repositoryURL` | `string` | The URL of the repository that hosts the service code                                        |
 | `serviceName`   | `string` | The name of the service inside the project configuration                                     |
 | `templateId`    | `string` | The identifier of the template in the Console marketplace from which the service was created |
+| `service`       | `object` | The service metadata (ideally, the whole service resource)                                   |
```

Comment:
This is the template resource more than the service. Should we change this?
Maybe service is more intended to the generated service, instead here it's the template used to generate the service